packages/mui-lab/src/Timeline/Timeline.types.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line no-restricted-imports -- importing types
+import { InternalStandardProps as StandardProps } from '@mui/material';
```

Comment:
The name was wrong (`tsx` instead of `ts`) so the props were considered external (maybe we could loosen the `implementedBySelfPropsFile` variable in `generateProptypes`